cli/src/app.rs

```diff
@@ -174,6 +174,7 @@ pub fn app<'a>() -> clap::Command<'a> {
                     .short('j')
                     .long("json")
                     .help("Produce output in json format (default: false)"),
+                Arg::new("filter").long("filter").value_name("filter").help(FILTER_ABOUT),
```

Comment:
I added the `--filter` flag to `phylum package`, because I see no reason why it wouldn't have this option. It lists issues so those can be easily filtered.